exporters/orch_tickets_exporter/orch_tickets_exporter.go

```diff
@@ -13,33 +13,64 @@ import (
 )
 
 var (
-	getRedeemedTicketsEndpoint = "https://stronk.rocks/api/livepeer/getAllRedeemTicketEvents"
+	winningTicketRedeemedEventsEndpoint = "https://api.thegraph.com/subgraphs/name/livepeer/arbitrum-one"
 )
 
-// ticketTransaction represents the structure of the ticket transaction field contained in the API response.
-type ticketTransaction struct {
-	Address         string
-	Amount          float64
-	TransactionHash string
-	BlockNumber     int
-	BlockTime       int
+// graphqlQuery represents the GraphQL query to fetch data from the GraphQL API.
+const graphqlQuery = `
+{
+	winningTicketRedeemedEvents(
+		where: {recipient: "0x5bdeedca9c6346b0ce6b17ffa8227a4dace37039"}
```

Comment:
Recipient should use `LIVEPEER_EXPORTER_ORCHESTRATOR_ADDRESS` ? Might want to cast it to lowercase as well, since graph queries only work for the lowercase version of a ETH address for some reason